extensions/data-plane/data-plane-http/src/test/java/org/eclipse/dataspaceconnector/dataplane/http/pipeline/HttpDataSinkFactoryTest.java

```diff
@@ -110,6 +110,34 @@ void verifyCreateAuthenticatingSource() throws InterruptedException, ExecutionEx
         verify(call).execute();
     }
 
+    @Test
+    void verifyCreateAdditionalHeaders() throws InterruptedException, ExecutionException, IOException {
+        var dataAddress = HttpDataAddress.Builder.newInstance()
+                .baseUrl("http://example.com")
+                .addAdditionalHeader("Content-Type", "application/test-octet-stream")
```

Comment:
`Content-Type` would fit better as an `HttpDataAddress` field, and then be used into the `StreamingRequestBody`, as it's not an additional header, more a mandatory one, and on okhttp the content type is attached to the request body directly.
It should have a default value of `application/octet-stream` to avoid breaking changes.

Reply:
Content-Type has it's own field now - in case it is added as additional header, it will not be taken into consideration